### PR TITLE
docs(popover): add popoverContext input demo

### LINKED ISSUE #3808: docs(popover): add popoverContext input demo
add demo where context will be used if popover is a template

Related to #3255

### PR DESCRIPTION
**Pull request by tarusin**
Add demo where context will be used if popover is a template
Closes #3808

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+popover/demos/popover-context/popover-context.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ng-template #popTemplate let-contextMessageLocal="contextMessage">{{ contextMessageLocal }}</ng-template>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+popover/demos/popover-context/popover-context.html]: that's too long 
` let-contextMessageLocal` -> ` let-message`

File: demo/src/app/components/+popover/demos/popover-context/popover-context.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-popover-context',
+  templateUrl: './popover-context.html'
+})
+export class DemoPopoverContextComponent {
+  context = {
+    contextMessage: 'Hello there!'
```
Comment (IlyaSurmay) [inline demo/src/app/components/+popover/demos/popover-context/popover-context.ts]: just `message`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

File: demo/src/app/components/+popover/demos/popover-context/popover-context.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<ng-template #popTemplate let-message="message">{{ message }}</ng-template>
+<button type="button" class="btn btn-primary"
+        [popover]="popTemplate" [popoverContext]="context">
+  Toggle context
```
Comment (IlyaSurmay) [inline demo/src/app/components/+popover/demos/popover-context/popover-context.html]: Open popover with custom context

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Not clear, why this demo called input demo, only template content displayed there. @tarusin ,please, clarify this moment
whyinput

**Comment by YevheniiaMazur** [comment]
Clarified, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**